cloudformation/dynamoDb.json

```diff
@@ -197,6 +197,46 @@
         }
       },
       "DeletionPolicy": "Retain"
+    },
+    "TrafficDriverRejectDBTable": {
+      "Type": "AWS::DynamoDB::Table",
+      "Properties": {
+        "AttributeDefinitions": [
+          {
+            "AttributeName": "campaignId",
+            "AttributeType": "S"
+          }, {
+            "AttributeName": "lineItemId",
+            "AttributeType": "N"
+          }
+        ],
+        "KeySchema": [
+          {
+            "AttributeName": "campaignId",
+            "KeyType": "HASH"
+          },{
+            "AttributeName": "lineItemId",
+            "KeyType": "RANGE"
+          }
+        ],
+        "ProvisionedThroughput": {
+          "ReadCapacityUnits": "1",
+          "WriteCapacityUnits": "1"
```

Comment:
I know times are tight but we can probably splurge on a bit more capacity. Setting it to 1 is probably a recipe for alarms firing and throttling on standard use. I'd suggest 5 at the minimum